vita3k/ngs/src/modules/atrac9.cpp

```diff
@@ -246,7 +246,7 @@ bool Atrac9Module::decode_more_data(KernelState &kern, const MemState &mem, cons
     }
 
     const int32_t sample_rate = data.parent->rack->system->sample_rate;
-    if (params->playback_scalar != 1 || static_cast<int>(round(params->playback_frequency)) != sample_rate) {
+    if (params->playback_scalar != 1.0f || static_cast<int>(round(params->playback_frequency)) != sample_rate) {
```

Comment:
I would also revert this change as it is not related to the bug. However, it is not as critical.